app/src/main/java/org/mozilla/fenix/library/history/HistoryFragment.kt

```diff
@@ -287,4 +298,10 @@ class HistoryFragment : Fragment(), CoroutineScope, BackHandler {
             HistoryFragmentDirections.actionHistoryFragmentToShareFragment(url = url, tabs = tabs?.toTypedArray())
         Navigation.findNavController(view!!).navigate(directions)
     }
+
+    private inline fun emitChange(producer: () -> HistoryChange) {
+        getManagedEmitter<HistoryChange>().onNext(producer())
+    }
 }
+
+private const val HISTORY_TIME_DAYS = 3L
```

Comment:
should we put this in a companion object in this class?

Reply:
@ekager I think so. I'm going to land this for now as I have to fix some bugs and I'll switch this in that PR

Reply:
Are there some reasons to put private compile-time constants like this into companion objects? Since files in Kotlin are valid visibility and semantics scope. :thinking: